Control coverage/Feature/FrontDoor.md

```diff
@@ -0,0 +1,66 @@
+# FrontDoor
+
+**Resource Type:** Microsoft.Network/frontDoor 
+
+<!-- TOC -->
+
+- [Azure_FrontDoor_NetSec_Enable_WAF_Configuration](#azure_frontdoor_netsec_enable_waf_configuration)
+
+
+<!-- /TOC -->
+<br/>
+
+___ 
+
+## Azure_FrontDoor_NetSec_Enable_WAF_Configuration 
+
+### Display Name 
+Front Door (Classic) should have Web Application Firewall configured
+
+### Rationale 
+Azure Web Application Firewall (WAF) on Azure Front Door provides centralized protection for your web applications. WAF defends your web services against common exploits & vulnerablities. It keeps your service highly available for your users and helps you meet compliance requirements.
+
+ ### Control Spec 
+
+> **Passed:** 
+> Web Application Firewall has been configured on Front Door </br>
+> and Configured WAF Policy mode must be Prevention only. </br> 
+> and Configured WAF Policy mode must be in Enabled State only. </br> 
+> 
+> **Failed:** 
+> WAF is not configured on Front Door. </br> 
+> or Configured WAF Policy mode is not Prevention. </br> 
+> or Configured WAF Policy mode is not in Enabled State. </br> 
+> 
+> **Error:** 
+> There was an error fetching WAF Configuration details of Front Door.
+> 
+### Recommendation 
+- **Azure Portal** 
+
+	 Use the Azure portal to configure WAF Policy on the Front Door.<br/>
```

Comment:
Can we give more details, how users can use Azure portal to remediate resources?
Some docs link might be helpful here?

Reply:
added doc link to add WAF Policy